src/utils/gitHubUtils.ts

```diff
@@ -157,3 +158,29 @@ export function isUser(orgData: gitHubOrgData | undefined): boolean {
     // if there's no orgData, just assume that it's a user (but this shouldn't happen)
     return orgData ? orgData.type === 'User' : true;
 }
+
+export function convertConclusionToVerb(conclusion: Conclusion): string {
+    switch (conclusion) {
```

Comment:
I think this can be cleaned up a bit if you use an enum instead of a type.  It'll look like this: https://github.com/microsoft/vscode-azurefunctions/blob/67c8ebc6b8931e6ab379dfe42c9c747de2c5b03e/src/tree/remoteProject/RemoteFunctionTreeItem.ts#L63-L73

Reply:
FWIW, I don't care about "enum" vs. "type". It's six of one and half a dozen of the other